python/edm4hep/datamodel.py

```diff
@@ -0,0 +1,14 @@
+import importlib
+import ROOT
+ROOT.gSystem.Load('libedm4hepDict.so')
```

Comment:
This returns either `0`, `1` or `-1`, where `-1` indicates a failure to load, that could be used to raise an `ImportError` with a message. If the load succeeded, `from ROOT import edm4hep` should work without problems and we wouldn't need the `importlib`

Reply:
Fixed but `importlib` is needed because even though you can do `from ROOT import edm4hep` the names of the classes don't exist until they are called explicitly so either you load all of them at the beginning or you load them dynamically like it's done here:
https://github.com/key4hep/EDM4hep/blob/python-bindings/python/edm4hep/datamodel.py#L17

Reply:
Actually no, it's possible to do it and `importlib` has been removed